engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
That is odd. The demo files that I have are different from the ones provided by Sierrahelp. The ones from Sierrahelp do include fonts, the ones I got don't. Also, the ones I got seem to be using an older 16-bit executable, which is consistent with SCI2/SCI2.1, but the ones in Sierrahelp have a 32-bit executable.

The demos from Sierrahelp have been modified, as the page states, so this could be the reason